bazel/plasma.bzl

```diff
@@ -0,0 +1,80 @@
+load("@com_github_google_flatbuffers//:build_defs.bzl", "flatbuffer_cc_library")
```

Comment:
Should this file be eventually moved to the plasma repo?

Reply:
absolutely!